src/palette/internal/palettecreator.cpp

```diff
@@ -1547,65 +1547,37 @@ PalettePtr PaletteCreator::newTextPalette(bool defaultPalette)
     rhm->setXmlText("B1");
     sp->appendElement(rhm, QT_TRANSLATE_NOOP("palette", "Rehearsal mark"));
 
-    auto legato = makeElement<PlayTechAnnotation>(gpaletteScore);
-    legato->setXmlText(QT_TRANSLATE_NOOP("palette", "legato"));
-    legato->setTechniqueType(PlayingTechniqueType::Legato);
-    sp->appendElement(legato, QT_TRANSLATE_NOOP("palette", "Legato"))->setElementTranslated(true);
-
-    auto pz = makeElement<PlayTechAnnotation>(gpaletteScore);
-    pz->setXmlText(QT_TRANSLATE_NOOP("palette", "pizz."));
-    pz->setTechniqueType(PlayingTechniqueType::Pizzicato);
-    sp->appendElement(pz, QT_TRANSLATE_NOOP("palette", "Pizzicato"))->setElementTranslated(true);
-
-    auto ar = makeElement<PlayTechAnnotation>(gpaletteScore);
-    ar->setXmlText(QT_TRANSLATE_NOOP("palette", "arco"));
-    ar->setTechniqueType(PlayingTechniqueType::Natural);
-    sp->appendElement(ar, QT_TRANSLATE_NOOP("palette", "Arco"))->setElementTranslated(true);
-
-    auto tm = makeElement<PlayTechAnnotation>(gpaletteScore);
-    tm->setTextStyleType(TextStyleType::EXPRESSION);
-    tm->setXmlText(QT_TRANSLATE_NOOP("palette", "tremolo"));
-    tm->setTechniqueType(PlayingTechniqueType::Tremolo);
-    sp->appendElement(tm, QT_TRANSLATE_NOOP("palette", "Tremolo"))->setElementTranslated(true);
-
-    auto mu = makeElement<PlayTechAnnotation>(gpaletteScore);
-    //: For brass and plucked string instruments: staff text that prescribes to use mute while playing, see https://en.wikipedia.org/wiki/Mute_(music)
-    mu->setXmlText(QT_TRANSLATE_NOOP("palette", "mute"));
-    mu->setTechniqueType(PlayingTechniqueType::Mute);
-    //: For brass and plucked string instruments: staff text that prescribes to use mute while playing, see https://en.wikipedia.org/wiki/Mute_(music)
```

Comment:
It might be nice to add these hints in `PLAY_TECH_TYPES` in `typesconv.cpp`, but for the rest this looks good to me now!